android/src/main/java/com/rnthreads/ThreadBaseReactPackage.java

```diff
@@ -30,23 +30,7 @@ public ThreadBaseReactPackage(ReactInstanceManager reactInstanceManager) {
 
     @Override
     public List<NativeModule> createNativeModules(ReactApplicationContext catalystApplicationContext) {
-        return Arrays.<NativeModule>asList(
-                // Core list
-                new AndroidInfoModule(catalystApplicationContext),
```

Comment:
So we don't want these packages anymore aside from the last one?

Reply:
That's right!

After the past couple of iterations on this fork, it seems we keep running into issues where there is a React Native core module being implicitly imported within the scope of the JavaScript thread, but we were lacking a corresponding instantiation on the `ThreadBaseReactPackage`.

I believe these missing implementations were being deferred through lazy loading in development builds, whereas when building for release, they were expected to be imported ahead of time.

Rather than exhaustively enabling each module, it turned out it was possible to inherit these from the current `ReactInstanceManager`:

```java
// react native runtime
threadPackages.addAll(getReactInstanceManager().getPackages());
```

https://github.com/ExodusMovement/react-native-threads/pull/10/commits/18095f6aeaccabf32cd13c38aa0a0bc379079741#diff-8f9883e78d06dae01be4c04473b1e322a3ae7400115cff85e6661546653ac809R84

So we can guarantee the existence of all the dependencies we need.